Artifacts/windows-vsts-build-agent/vsts-agent-install.ps1

```diff
@@ -0,0 +1,122 @@
+# Downloads the Visual Studio Online Build Agent, installs on the new machine, registers with the Visual
+# Studio Online account, and adds to the specified build agent pool
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
```

Comment:
Added ValidateNotNullOrEmpty. Not sure how to use ValidatePattern with the negative tests I have.